src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.VisualBasic.xaml

```diff
@@ -1,6 +1,375 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
 <Rule Name="Build"
-      OverrideMode="Extend"
+      OverrideMode="Replace"
+      Description="Specifies properties that control how the project builds."
+      DisplayName="Compile"
+      PageTemplate="generic"
+      Order="200"
       xmlns="http://schemas.microsoft.com/build/2009/properties">
 
+  <Rule.Categories>
+    <Category Name="General"
+              DisplayName="General" />
+
+    <Category Name="CompileOptions"
+              DisplayName="Compile Options" />
+
+    <Category Name="WarningConfigurations"
+              DisplayName="Warning configurations" />
+
+    <Category Name="Events"
+              Description="Configures custom events that run before and after build."
+              DisplayName="Events" />
+
+    <Category Name="Advanced"
+              DisplayName="Advanced"
+              Description="Advanced settings for the application." />
+  </Rule.Categories>
+
+  <Rule.DataSource>
+    <DataSource Persistence="ProjectFileWithInterception"
+                HasConfigurationCondition="True" />
+  </Rule.DataSource>
+
+  <StringProperty Name="Output path"
```

Comment:
Should we also bring the `Browse` button for this property?
![image](https://user-images.githubusercontent.com/8518253/167503676-40a617e5-c8d2-4d07-8026-76ed5c81c49c.png)

Reply:
I would have expected `Subtype="directory"` to do that. It seems to work for other properties (i.e. base output path).

Reply:
Yes, I have `Subtype="directory"` and the "Browse" button is present.